collect-settings.rst

```diff
@@ -97,16 +97,14 @@ Form Management Settings
 
 - :guilabel:`Default to finalized` When enabled, forms are automatically finalized upon reaching the end of the form. You can opt out of finalizing any form when completing it.
 - :guilabel:`Constraint processing` used to select when response constraints are validated: After each swipe, or at finalization.
-- :guilabel:`High res video` enables high-resolution recordings.
-- :guilabel:`Image size` is used to set an image size for all the widgets.There are five size options available as shown below:
-
-.. image:: /img/collect-settings/image-settings.png
-  :alt: Image settings 
-  :class: device-screen-vertical
-
-.. versionadded:: 1.11.0*
-   :guilabel:`Image size` feature will be added in *Collect v1.11.0* (upcoming release).
-  
+- :guilabel:`High res video` enable or disable high-resolution video recordings if supported by the video application used.
+- :guilabel:`Image size` (v1.11.0+) specifies the maximum number of pixels for the long edge of all images added to forms. Images are scaled down immediately after being added. This setting can be overwridden at the form question level. There are five size options available:
+
+  - :guilabel:`Original size from camera (default)`: images are unchanged when added to a form. Used when images must contain a lot of detail or when the internet connection used to send submissions is fast.
+  - :guilabel:`Very small (640px)`: used when images don't need to be detailed and the internet connection used to send submissions is slow.
+  - :guilabel:`Small (1024px)`: sufficiently detailed for most on-screen viewing but too small for printing.
+  - :guilabel:`Medium (2048px)`: sufficiently detailed for most uses including printing.
+  - :guilabel:`Large (3072px)`: used when a lot of detail is needed.
```

Comment:
All descriptions should follow a consistent pattern. Each line should start as used when images....
> used when images need to be sufficiently detailed....
> used when images need a lot of detailing...

Reply:
I'm usually a big fan of parallel constructions but in this case I thought it might be easier to contrast the choices if only the differentiating information was provided. I don't feel strongly about it, though. @adammichaelwood?

Reply:
@ankita240796 is right in principle. But since we're writing tech docs and not poetry, I think @lognaturel's is more immediately clear for our users.

Reply:
We can do the ODK poetry next. 😉